lib/bambora/v1/payment_resource.rb

```diff
@@ -9,9 +9,25 @@ class PaymentResource
       # Docs: https://dev.na.bambora.com/docs/references/payment_APIs/
       #       https://dev.na.bambora.com/docs/references/payment_SDKs/take_payments/?shell#
       # Endpoint: https://api.na.bambora.com/v1/payments
-      def initialize(client:, sub_path:)
+      def initialize(client:)
         @client = client
-        @sub_path = sub_path
+        @sub_path = '/v1/payments'
+      end
+
+      def make_payment(payment_data)
+        @client.request(method: :post, path: @sub_path, body: payment_data)
+      end
+
+      def make_payment_with_payment_profile(customer_code:, amount:, card_id: 1, complete: false)
```

Comment:
Hey, I know we've gone back and forth on this a lot, but this looks like a magic number again. Perhaps some documentation above this method would help clarify the usage here. Perhaps we can do that in a follow up PR.